### ISSUE TITLE
while training,dice_loss and iou_score are always nan

### ISSUE THREAD

**Issue by HaoranMaier** [dvalex/daunet#4]
I trained this model on cracks500(first stage),but while training, the loss is almost unchanged, and the dice_loss and iou_score are always nan ,i don't know how to fix it.

**Comment by HaoranMaier** [comment]
I updated the vesion of tensorflow and then solved this problem.
